cluster/ci/config/prow/openshift/artifact-uploader.yaml

```diff
@@ -39,6 +39,15 @@ items:
           - name: gcs-credentials
             mountPath: /secrets/gcs
             readOnly: true
+          resources:
+            requests:
+              memory: "2Pi"
+              cpu: "100m"
+              ephemeral-storage: "30Mi"
+            limits:
+              memory: "4Pi"
```

Comment:
What is `Pi`?

Reply:
According to this [doc](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory), 

> Limits and requests for memory are measured in bytes. You can express memory as a plain integer or as a fixed-point integer using one of these suffixes: E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki. F

Reply:
So are you specifying petabytes of memory?

Reply:
I tried to adjust the request regarding the grafana dashboard. It seems the I messed it up with the conversion. Let me fix that.